.github/workflows/python.yml

```diff
@@ -8,6 +8,13 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
+      - name: Create venv
```

Comment:
Is this the correct way of doing it, or should we use something like this action for it?

https://github.com/galaxyproject/iwc/blob/main/.github/workflows/setup.yml#L63C1-L65C53

@arash77 you know these things correct?

Reply:
Both ways will work to set up a virtual environment, but you're right that using [actions/setup-python](https://github.com/actions/setup-python) is better. It handles the setup automatically and makes sure everything is ready without needing to manually activate the virtual environment.